src/headerData.json

```diff
@@ -120,6 +120,10 @@
       "remainingCredits": "1422100",
       "buttonText": "UPGRADE"
     },
+    "clientManager": {
+      "logo": "https://via.placeholder.com/180x50",
```

Comment:
As I've told you, it would be very nice to have our own placehoder here. Whenever you can tal this with @santiagopaolucci

Reply:
Los placeholder quedaran del lado de la librería según lo conversado con @santiagopaolucci